### PR TITLE
If 545 adjustments for internationalization

### PR DESCRIPTION
**Pull request by florian-paul**

### CODE REVIEW

**Comment by mattmikulina** [comment]
Task linked: [IF-545 Adjustments for internationalization](https://app.clickup.com/t/2177764/IF-545)

File: fusionfrontend/src/app/ecosystem/components/content/field-list/field-list.component.html [hunk 1]
```diff
@@ -58,6 +58,10 @@ <h3 class="table-header">{{ displayedFields?.length | i18nPlural: titleMapping }
 </p-table>
 
 
-<p-confirmDialog acceptButtonStyleClass="if-button button-secondary" class="delete-confirm-dialog"
-                 rejectButtonStyleClass="if-button button-secondary"></p-confirmDialog>
+<p-confirmDialog #cd class="delete-confirm-dialog">
```
Comment (jsylupp) [inline fusionfrontend/src/app/ecosystem/components/content/field-list/field-list.component.html:61]: Consider extracting component

File: fusionfrontend/src/app/shared/components/ui/header/header.component.scss [hunk 2]
```diff
@@ -414,31 +414,54 @@ header {
   }
 }
 
+:host ::ng-deep p-dropdown .p-dropdown {
+  //vertical-align: -webkit-baseline-middle;
```
Comment (jsylupp) [inline fusionfrontend/src/app/shared/components/ui/header/header.component.scss]: remove comment

File: fusionfrontend/src/assets/i18n/de.json [hunk 3]
```diff
@@ -11,11 +11,13 @@
 				"EDIT": "Bearbeiten",
 				"FINISH": "Abschließen",
 				"NEXT": "Weiter",
+        "NO": "Nein",
```
Comment (jsylupp) [inline fusionfrontend/src/assets/i18n/de.json:14]: alignment

File: fusionfrontend/src/assets/i18n/en.json [hunk 4]
```diff
@@ -11,11 +11,13 @@
 				"EDIT": "Edit",
 				"FINISH": "Finish",
 				"NEXT": "Next",
+        "NO": "No",
```
Comment (jsylupp) [inline fusionfrontend/src/assets/i18n/en.json:14]: alignment

File: fusionfrontend/src/assets/i18n/fr.json [hunk 5]
```diff
@@ -11,12 +11,14 @@
 				"EDIT": "edit",
 				"FINISH": "Finition",
 				"NEXT": "Suivant",
-				"ONBOARD": "A bord",
+        "NO": "Non",
```
Comment (jsylupp) [inline fusionfrontend/src/assets/i18n/fr.json:14]: alignment

**Review by jsylupp — CHANGES_REQUESTED** [review]
Well done! Only little suggestions

**Pull request merged**